XamlControlsGallery/ControlPages/PipsPagerPage.xaml

```diff
@@ -0,0 +1,85 @@
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <ComboBox x:Name="OrientationComboBox" Header="Orientation" SelectionChanged="OrientationComboBox_SelectionChanged" SelectedValue="Horizontal">
+                        <x:String>Horizontal</x:String>
+                        <x:String>Vertical</x:String>
+                    </ComboBox>
+                    <ComboBox x:Name="PrevButtonComboBox" Header="Previous Button Visibility" SelectionChanged="PrevButtonComboBox_SelectionChanged" SelectedValue="Visible">
+                        <x:String>Visible</x:String>
+                        <x:String>VisibleOnPointerOver</x:String>
+                        <x:String>Collapsed</x:String>
+                    </ComboBox>
+                    <ComboBox x:Name="NextButtonComboBox" Header="Next Button Visibility" SelectionChanged="NextButtonComboBox_SelectionChanged" SelectedValue="Visible">
+                        <x:String>Visible</x:String>
+                        <x:String>VisibleOnPointerOver</x:String>
+                        <x:String>Collapsed</x:String>
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;muxc:PipsPager
+    Orientation="$(Orientation)"
+    PreviousButtonVisibility="$(PrevButton)"
+    NextButtonVisibility="$(NextButton)" /&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Orientation" Value="{x:Bind OrientationComboBox.SelectedValue, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="PrevButton" Value="{x:Bind PrevButtonComboBox.SelectedValue, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="NextButton" Value="{x:Bind NextButtonComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example2" HeaderText="PipsPager integrated with a FlipView">
+            <local:ControlExample.Example>
+                <StackPanel>
+                    <FlipView x:Name="Gallery" MaxWidth="400" Height="270" ItemsSource="{x:Bind Pictures}">
+                        <FlipView.ItemTemplate>
+                            <DataTemplate>
+                                <Image Source="{Binding}"/>
+                            </DataTemplate>
+                        </FlipView.ItemTemplate>
+                    </FlipView>
+                    <muxc:PipsPager x:Name="FlipViewPipsPager"
+                              Margin="140, 10, 0, 0"
+                              NumberOfPages="{x:Bind Pictures.Count}"
+                              SelectedPageIndex="{Binding ElementName=Gallery, Path=SelectedIndex, Mode=TwoWay}" />
```

Comment:
General question: this uses old-style Binding instead of x:Bind. Is it possible to use x:Bind with PipsPager?